inspirehep/dojson/hep/model.py

```diff
@@ -26,4 +26,19 @@
 from dojson import utils
 
 hep = Overdo()
+
+
+@hep.over('book', '^773..')
```

Comment:
Yeah, the order is not guaranteed. In the past we would add the extra stuff directly in the function (in this case collections), but it is not always nice as collections is now shared between all dojsons in `base.py`.

@jmartinm Another possibly less "magic" way could be to use inheritence to subclass the `Overdo` class in the hep model and there override the `do()` function to call the super version and then do our code.